### DISCUSSION TITLE
Can swc use existing tsconfig.json for configuration?

### DISCUSSION THREAD

**Discussion by winston0410** [swc-project/swc#4662] [Q&A] [answered]
There are some overlapping configuration between swc and tsconfig.json. Can it be picked up automatically?

**Comment by kwonoj** [comment, marked as answer]
This isn't technically impossible, but we're trying to be careful to implicitly interop between configs can cause side effects, as well as make explicit connect between config values across different tools other than swc.

**Comment by ValentinGurkov** [comment]
Sorry for picking this back up - do you support the use of `tsconfig,json`? When there's no `swc` file? I'm trying to plan for a migration

**Reply by kwonoj** [reply]
swc config uses .swcrc only.

**Reply by ValentinGurkov** [reply]
Thank you. I will just have to learn the equivalent configurations in .swcrc
